assets/styles/style.css

```diff
@@ -1,36 +1,69 @@
 .topic h1 {
-	font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
-	font-size: 30px;
-	margin-bottom: 20px;
+  font-family: "Times New Roman", Times, serif;
+  font-size: 30px;
+  margin-bottom: 20px;
 }
 
 #background {
-	background-image: image("coder.jpg");
-	background-size: cover;
-	background-position: center;
+  background: linear-gradient(#7777, #7890), url("../images/coder.png");
+  background-size: cover;
+  background-position: center;
+  color: #fff;
 }
```

Comment:
Your image is shaking with your form. On the original it's only the form that shakes the image is static pay attention to that please

Reply:
How do you think I should go about that?